diff --git livox-sdk/CMakeLists.txt b/CMakeLists.txt
index 05f84c5..8c1bf05 100644
--- livox-sdk/CMakeLists.txt
+++ livox-sdk_fixed/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.20)
 
 project(livox_sdk2)
 
diff --git livox-sdk/samples/CMakeLists.txt b/samples/CMakeLists.txt
index 022dda1..9adb9f7 100644
--- livox-sdk/samples/CMakeLists.txt
+++ livox-sdk_fixed/samples/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.20)
 
 project(livox_sdk2)
 
@@ -16,11 +16,11 @@ if (UNIX)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 endif(UNIX)
 
-add_subdirectory(livox_lidar_quick_start)
-add_subdirectory(multi_lidars_upgrade)
-add_subdirectory(logger)
-add_subdirectory(debug_point_cloud)
-add_subdirectory(lidar_cmd_observer)
-add_subdirectory(livox_lidar_rmc_time_sync)
-add_subdirectory(livox_lidar_ip_set)
-add_subdirectory(livox_lidar_info_get)
+#add_subdirectory(livox_lidar_quick_start)
+#add_subdirectory(multi_lidars_upgrade)
+#add_subdirectory(logger)
+#add_subdirectory(debug_point_cloud)
+#add_subdirectory(lidar_cmd_observer)
+#add_subdirectory(livox_lidar_rmc_time_sync)
+#add_subdirectory(livox_lidar_ip_set)
+#add_subdirectory(livox_lidar_info_get) 
diff --git livox-sdk/sdk_core/CMakeLists.txt b/sdk_core/CMakeLists.txt
index ee410c1..dac94af 100644
--- livox-sdk/sdk_core/CMakeLists.txt
+++ livox-sdk_fixed/sdk_core/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.20)
 
 set(SDK_LIBRARY_STATIC livox_lidar_sdk_static)
 set(SDK_LIBRARY_SHARED livox_lidar_sdk_shared)
diff --git livox-sdk/sdk_core/comm/define.h b/sdk_core/comm/define.h
index 01b1c2a..f12a76f 100644
--- livox-sdk/sdk_core/comm/define.h
+++ livox-sdk_fixed/sdk_core/comm/define.h
@@ -33,6 +33,7 @@
 #include <cstdint>
 #include <atomic>
 
+#include <cstdint>
 #include "livox_lidar_def.h"
 
 namespace livox {
